src/components/login.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default class Login extends React.Component {
+  state = {
+    input: '',
+    submit: false
+  };
+
+  onKeyPress = event => {
+    const target = event.target;
+    const valid = target.checkValidity();
+
+    this.setState({ input: event.target.value, submit: valid });
+  };
+
+  onSubmit = () => {
+    this.props.handleSubmit(this.state.input);
+  };
+
+  render() {
+    return (
+      <div className="login">
+        <label htmlFor="login">
+          Enter your phone number
+          <input
+            value={this.state.value}
+            type="tel"
+            id="login"
+            placeholder="Enter your phone number"
+            pattern="[0-9]{3}-?[0-9]{3}-?[0-9]{3}"
```

Comment:
Change pattern to +32 and add a selection box to the front of number so we have [+32] [input rest here]